documentation/modules/creating-vddk-image.adoc

```diff
@@ -30,8 +30,14 @@ Storing the VDDK image in a public registry might violate the VMware license ter
 $ mkdir /tmp/<dir_name> && cd /tmp/<dir_name>
 ----
 
-. In a browser, navigate to the link:https://developer.vmware.com/web/sdk/7.0/vddk[VMware VDDK version 7 download page].
-. Select version 7.0.3.2 and click *Download*.
+. In a browser, navigate to the link:https://developer.vmware.com/web/sdk/8.0/vddk[VMware VDDK version 8 download page].
+. Select version 8.0.1 and click *Download*.
```

Comment:
@qiyuann please double check that this is the version that you're going to test with